### PR TITLE
fix(project): change operators order

### PR DESCRIPTION
**Pull request by jeremyvergnas**
Makes more sense to have operator `contains` selected by default.

### CODE REVIEW

**Comment by RWOverdijk** [comment]
Maybe it makes most sense to make it configurable? This single change would require a major version bump for a simple swap. Making a config is really simple so it might be worth doing that.

**Comment by jeremyvergnas** [comment]
@RWOverdijk Done :)

File: src/filter.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
-import {customElement, bindable, bindingMode} from 'aurelia-framework';
+import {inject, customElement, bindable, bindingMode} from 'aurelia-framework';
 import {resolvedView}                         from 'aurelia-view-manager';
 import {CriteriaBuilder}                      from './criteriaBuilder';
+import {Configuration}                        from 'aurelia-config';
 
 @customElement('filter')
 @resolvedView('spoonx/filter', 'filter')
+@inject(Configuration.of('aurelia-filter'))
```
Comment (VMBindraban) [inline src/filter.js:8]: Add aurelia-config to `package.json`.

Comment (jeremyvergnas) [reply src/filter.js:8]: I thought `aurelia-config` was part of `spoonx-tools`, just checked, and it is not the case... 

Then I guess we have the same issue for `aurelia-form`.

Comment (jeremyvergnas) [reply src/filter.js:8]: @RWOverdijk ⬆️

**Pull request merged**